pkg/querier/tail.go

```diff
@@ -14,12 +14,22 @@ import (
 
 const (
 	// if we are not seeing any response from ingester, how long do we want to wait by going into sleep
-	nextEntryWait = time.Second / 2
+	tailerWaitEntryThrottle = time.Second / 2
```

Comment:
I think we should move this to `pkg/querier/querier.go` since it is used in that file.

Reply:
Definitely. I was in doubt as well. You confirmed my doubt.